yammbs/analysis.py

```diff
@@ -241,15 +235,8 @@ def get_internal_coordinates(
                     ),
                 )
 
-            key = tuple(int(index) for index in key)
-
             internal_coordinates[label].update(
-                {
-                    key: (
-                        internal_coordinate.value(reference),
-                        internal_coordinate.value(target),
-                    ),
-                },
+                {tuple(int(index) for index in key): float(internal_coordinate.value(coordinates))},
```

Comment:
This is causing https://github.com/openforcefield/yammbs/issues/114#issuecomment-2790808697 which has me wondering if the coordinates passed to `PrimitiveInternalCoordinates` and/or `geometric.molecule.Molecule` cause some sort of auto-magic filtering